src/Options/index.js

```diff
@@ -61,8 +62,8 @@ export interface ParseServerOptions {
   appName: ?string;
   /* Add headers to Access-Control-Allow-Headers */
   allowHeaders: ?(string[]);
-  /* Sets the origin to Access-Control-Allow-Origin */
-  allowOrigin: ?string;
+  /* Sets the origin to Access-Control-Allow-Origin. Can be a string for a single origin or a comma separated string or array for multiple */
```

Comment:
We will likely remove the "comma separated string" option in a future release; this can create parsing issues which we have seen in the past. We will likely also remove the option of single string in a future release to only allow an array of strings and rename this option `allowOrigins` for better semantics.

```suggestion
  /* Sets origins for Access-Control-Allow-Origin. This can be a string for a single origin or an array of strings for multiple origins. */
```